app/controllers/admin_controller.rb

```diff
@@ -1,15 +1,33 @@
 class AdminController < ApplicationController
   before_action :authenticate_admin!
 
-  def new
+  def newUser
+    render :template => admin_user_new_path
   end
 
-  def create
+  def createUser
     case params['commit']
     when "Submit CSV"
       redirect_to createUserFromAdminCsv(params)
     else
       redirect_to createUserFromAdminForm(params)
     end
   end
+
+  def newEvent
```

Comment:
newEvent should be under an Events controller, where we authenticate if the user is an admin before actually calling it.